sql/moz-fx-data-shared-prod/firefox_ios_derived/funnel_ios_v1/query.sql

```diff
@@ -0,0 +1,76 @@
+WITH views_data AS (
+  SELECT
+    `date`,
+    territory AS country_name,
+    SUM(impressions_unique_device) AS views,
+  FROM
+    app_store.firefox_app_store_territory_source_type_report
+  WHERE
+    `date` >= '2022-01-01'
+  GROUP BY
+    `date`,
+    country_name
+),
+downloads_data AS (
+  SELECT
+    `date`,
+    territory AS country_name,
+    SUM(total_downloads) AS downloads,
+  FROM
+    app_store.firefox_downloads_territory_source_type_report
+  WHERE
+    `date` >= '2022-01-01'
+    AND source_type <> 'Institutional Purchase'
+  GROUP BY
+    `date`,
+    country_name
+),
+store_stats AS (
+  SELECT
+    DATE(`date`) AS `date`,
+    code AS country,
+    views,
+    downloads,
+  FROM
+    views_data
+  JOIN
+    downloads_data
+  USING
+    (`date`, country_name)
+  JOIN
+    static.country_codes_v1
+  ON
+    country_name = name
+),
+activations AS (
+  SELECT
+    first_seen_date AS `date`,
+    country,
+    SUM(new_profile) AS new_profiles,
+    SUM(activated) AS activations,
+  FROM
+    firefox_ios.new_profile_activation
+  WHERE
+    submission_date >= '2022-01-01'
+    AND first_seen_date >= '2022-01-01'
+    AND NOT (
+      app_display_version = '107.2'
+      AND submission_date >= '2023-02-01'
+    ) -- incident filter  #TODO: is there an additional ticket or bug we can reference here?
```

Comment:
@bani Do we have a relevant bug or issue we could link here so that if needed we can easily find more context in the future?

Reply:
@fbertsch do you maybe know more about this issue and which bug we could provide here for additional context for this extra piece of logic?